dm/worker/config.go

```diff
@@ -150,6 +151,15 @@ func (c *Config) Parse(arguments []string) error {
 		return terror.ErrWorkerInvalidFlag.Generate(c.flagSet.Arg(0))
 	}
 
+	err = log.InitLogger(&log.Config{
```

Comment:
In fact, I don't think it's a good pattern to initialized the logger in `Config.adjust`, it's so tricky and the caller of `adjust` often does not know it will affect the logger.

if you just want to log something when `c.Name == "" {`, I prefer to use `fmt.Println` instead.